From 61ef5e7e206db0477ab5b9aa47fcbb92c15db1ec Mon Sep 17 00:00:00 2001
From: Yahya Toubali <yahya@yahyatoubali.me>
Date: Sat, 18 Jul 2026 17:56:13 +0100
Subject: [PATCH 06535/15333] Documentation: proc: fix repeated word 'page'

Remove the duplicate 'page' in the sentence describing large page
allocation accounting. The extra word makes the description confusing.

Signed-off-by: Yahya Toubali <yahya@yahyatoubali.me>
Signed-off-by: Jonathan Corbet <corbet@lwn.net>
Message-ID: <20260718165613.1923550-1-yahya@yahyatoubali.me>
---
 Documentation/filesystems/proc.rst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Documentation/filesystems/proc.rst b/Documentation/filesystems/proc.rst
index 2ccd5b2dfdd3..d22ac0addd42 100644
--- a/Documentation/filesystems/proc.rst
+++ b/Documentation/filesystems/proc.rst
@@ -513,7 +513,7 @@ In some kernel configurations, the semantics of pages part of a larger
 allocation (e.g., THP) can differ: a page is accounted as "private" if all
 pages part of the corresponding large allocation are *certainly* mapped in the
 same process, even if the page is mapped multiple times in that process. A
-page is accounted as "shared" if any page page of the larger allocation
+page is accounted as "shared" if any page of the larger allocation
 is *maybe* mapped in a different process. In some cases, a large allocation
 might be treated as "maybe mapped by multiple processes" even though this
 is no longer the case.
-- 
2.55.0

